HealthcareSystem/PatientWebApp/wwwroot/css/style.css

```diff
@@ -3,11 +3,11 @@
     width: 100%;
 }
 
-#search_btn {
-    width: 10%;
-    margin-left: 45%;
-    background-color: #33adff;
+.my_btn {
     color: white;
+    background-color: rgb(43, 110, 138);
+    width: 110px;
+    height: 38px;
 }
```

Comment:
### Comment

We are a company and we should have some documentation. This 'my_btn' is what? Where is it used, and why does it look as to how it looks?  

### Proposal
  - think about people who will read your code after you write it
  - make a more descriptive name
  - write documentation above
  - Loke at something like [this](https://webdesign.tutsplus.com/articles/css-documentation-best-practices--cms-30139)